utils/PortManagerServer.ts

```diff
@@ -37,29 +44,23 @@ class _PortManagerServer {
     this.app.use(cors());
     this.setupRoutes();
 
-    try {
-      this.server = await this.listen();
-      logger.debug(this.server);
-    } catch (e) {
-      if (isSystemError(e) && e.code === 'EADDRINUSE') {
-        throw new Error(
-          i18n(`${i18nKey}.errors.portInUse`, {
-            port: PORT_MANAGER_SERVER_PORT,
-          }),
-          { cause: e }
-        );
-      }
-      throw e;
```

Comment:
This error was never getting thrown, see my [other comment](https://github.com/HubSpot/hubspot-local-dev-lib/pull/334/files#r2504864351).